### ISSUE TITLE
with uiNamespace do is incorrect by linter

### ISSUE THREAD

**Issue by CreepPork** [SkaceKamen/vscode-sqflint#18]
```sqf
with uiNamespace do {call compile format ["[""LOADED"", controlNull] call (missionNamespace getVariable ""%1"");", _resource_fnc]};
```
Is detected wrong by the linter and displays as an error where in truth it is correct.

**Comment by Wyqer** [comment]
Don't know what you want to mention exactly.
Which one you use?
So your used tool is mentioning an error where no error is?

**Comment by SkaceKamen** [comment]
Will look into it

**Comment by SkaceKamen** [comment]
Should be fixed in 0.8.4
